back/Gemfile

```diff
@@ -122,7 +122,7 @@ gem 'premailer-rails', '~> 1.10.3'
 gem 'rest-client'
 gem 'rgeo-geojson'
 gem 'rubyzip', '~> 1.3.0'
-gem 'nokogiri', '>= 1.13.9'
+gem 'nokogiri', '>= 1.13.10'
```

Comment:
Not quite sure why these changes are included here, since I have merged them into master and then into the CL-1849/image-dimensions branch.

In any case, this change and the related changes in `Gemfile.lock` are OK.

Reply:
I guess sometimes Gemfile and Gemfile.lock changes may be made because we use volumes. Probably, if you want to get rid of them, you can:
1. Stop `docker-compose up`
2. Checkout these changes
3. Run `docker-compose up`